pipeline-runner/init.R

```diff
@@ -303,7 +309,6 @@ call_data_processing <- function(task_name, input, pipeline_config) {
 # reimport tryCatchLog & initialize states again.
 #
 start_heartbeat <- function(task_token, aws_config) {
-    library(tryCatchLog)
```

Comment:
this is already imported at the top of the file. There is no point importing it again